packages/next/src/server/next-server.ts

```diff
@@ -193,9 +193,10 @@ export default class NextNodeServer extends BaseServer<
     this.isDev = isDev
     this.sriEnabled = Boolean(options.conf.experimental?.sri?.algorithm)
 
-    const isTurbopackBuild = this.isTurbopackBuild()
-
     if (!isDev) {
+      const isTurbopackBuild = require(
+        join(this.distDir, SERVER_FILES_MANIFEST)
+      ).turbopack
```

Comment:
We don't need to read this file during server runtime if we put it on the config as noted above